pkgs/shells/fish/plugins/plugin-sudope.nix

```
@@ -0,0 +1,26 @@
+{
+  lib,
+  buildFishPlugin,
+  fetchFromGitHub,
+  unstableGitUpdater,
+}:
+buildFishPlugin {
+  pname = "plugin-sudope";
+  version = "unstable-2021-04-11";
```

Unstable versions should start with `0` (see [Package Naming](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming))

```suggestion
  version = "0-unstable-2021-04-11";
```